src/apps/workspaces/service/user_access.py

```diff
@@ -49,15 +51,33 @@ async def get_user_workspaces(self) -> list[UserWorkspace]:
         user_ids.append(self._user_id)
 
         workspaces = await UserWorkspaceCRUD(self.session).get_by_ids(user_ids)
-        return [UserWorkspace.from_orm(workspace) for workspace in workspaces]
+        return [
+            UserWorkspace(
+                user_id=workspace.user_id,
+                workspace_name=workspace.workspace_name,
+                integrations=[Integration.parse_obj(integration) for integration in json.loads(workspace.integrations)]
```

Comment:
Why do we need to parse json? Is it stores as a string with json array or just json array? Could you provide an example of the value?

Reply:
Yes, you're right, can be saved as json array instead of json string. If saved as json array, no need to parse.